CHANGELOG/preview.md

```diff
@@ -1,5 +1,98 @@
 # Preview Changelog
 
+## [7.6.0-preview.3]
+
+### General Cmdlet Updates and Fixes
+
+- Add completion single/double quote support for `-Noun` parameter (#24977) (Thanks @ArmaanMcleod!)
+- Remove trailing space from event source name (#24192) (Thanks @MartinGC94!)
```

Comment:
@daxian-dbw @SteveL-MSFT Should we tag this a breaking, even though it is bucket 3?

Reply:
I agree with tagging this as a breaking change in the change log.